.github/workflows/cheat-client-hashes.yml

```
@@ -0,0 +1,27 @@
+name: cheat-client-hashes
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+      branches:
+        - master
```

```suggestion
on:
  schedule:
    - cron: '0 0 * * *'
      branches:
        - master
  workflow_dispatch:
```
Maybe also create manual trigger (button in github actions page) just in case.